packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3377,13 +3371,13 @@ export class ContainerRuntime
 					contents: JSON.stringify(idAllocationMessage),
 					referenceSequenceNumber: this.deltaManager.lastSequenceNumber,
 					metadata: undefined,
-					localOpMetadata: this.idCompressor?.serialize(true),
+					localOpMetadata: undefined,
 					type: ContainerMessageType.IdAllocation,
 				};
 			}
 
 			if (idAllocationBatchMessage !== undefined) {
-				this.outbox.submit(idAllocationBatchMessage);
+				this.outbox.submitIdAllocation(idAllocationBatchMessage);
```

Comment:
nit: I don't think you need this `idAllocationBatchMessage` and could move the `submitIdAllocation(...)` call into the `if` scope above